src/rust/engine/fs/src/store.rs

```diff
@@ -2161,11 +2161,11 @@ mod tests {
   /// Create a StubCas with a file and a directory inside.
   ///
   pub fn new_cas(chunk_size_bytes: usize) -> StubCAS {
-    StubCAS::with_content(
-      chunk_size_bytes as i64,
```

Comment:
Is the change of this to usize related to the rest of the PR?
I'm guessing it's there because there's no reason for it to not be a usize, but I might be missing something.
Even if it's not directly related, I'm not against it being in this PR.

Reply:
It's related insofar as the reason this was an i64 before was so that we could use the magic value -1 to mean "always error". The reason we did that was because adding more constructor parameters was hassle, so magic values were easier. Now that new constructor parameters are cheap, we can be explicit (use a named bool) rather than implicit (use a magic value of another field) :)